xbmc/GUIInfoManager.cpp

```diff
@@ -1704,6 +1704,7 @@ const infomap system_labels[] =  {{ "hasnetwork",       SYSTEM_ETHERNET_LINK_ACT
 ///                  _boolean_,
 ///     @return **True** if the specified addon is installed on the system.
 ///     @param id - the addon id
+///     @skinning_v19 **[Boolean Condition Updated]** \link System_HasAddon `System.HasAddon(id)`\endlink
```

Comment:
In this case add also a new funktion like `System.HasEnabledAddon(id)`, on some places only the wanted enabled is request.

Thats also why this before given only the enabled addons.

Reply:
we already do have that one, it was added 2 weeks ago:
https://github.com/xbmc/xbmc/pull/13762